cypress/integration/tables.ts

```diff
@@ -0,0 +1,9 @@
+describe('table example', () => {
+  it('table tag rendered', () => {
+    cy.visit('examples/tables')
```

Comment:
Put this in a `beforeEach`, as every test in this file will be for the tables example (even though there's exactly one for now. Do the same with all `visit`s in a test (unless you have to `visit` multiple pages in a given test for some reason.)